lib/phoenix_live_view/channel.ex

```diff
@@ -344,7 +344,7 @@ defmodule Phoenix.LiveView.Channel do
   end
 
   def handle_info({:phoenix_live_reload, _topic, _changed_file}, %{socket: socket} = state) do
-    Phoenix.CodeReloader.reload(socket.endpoint)
+    socket.private.reloader.(socket.endpoint)
```

Comment:
let's just namespace the key and we can ship it IMO:

```suggestion
    socket.private.phoenix_reloader.(socket.endpoint)
```